pom.xml

```diff
@@ -44,62 +44,34 @@
    <dependencyManagement>
       <dependencies>
          <dependency>
-            <groupId>org.wildfly.swarm</groupId>
+            <groupId>io.thorntail</groupId>
             <artifactId>bom</artifactId>
-            <version>${version.wildfly.swarm}</version>
+            <version>${version.thorntail}</version>
             <type>pom</type>
             <scope>import</scope>
          </dependency>
 
          <dependency>
-            <groupId>org.wildfly.swarm</groupId>
-            <artifactId>bom-certified</artifactId>
-            <version>${version.wildfly.swarm}</version>
-            <type>pom</type>
-            <scope>import</scope>
```

Comment:
We shouldn't really remove `bom-certified`, that's where we get Arquillian from.

Reply:
Oh ok, this what was done in community but really affects prod. Gonna change that

Reply:
Yea community doesn't need `bom-certified` (it actually doesn't even exist in community releases).